great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -228,7 +228,7 @@ def _init_rule_domain_builder(
         return domain_builder
 
     @usage_statistics_enabled_method(
```

Comment:
@cdkini can you update the signature of `usage_statistics_enabled_method` to expect the `UsageStatsEvents` type?